packages/domains-table/src/domains-table/style.scss

```diff
@@ -284,7 +284,6 @@
 .domains-table-mobile-cards {
 	overflow-y: auto;
 	max-height: calc(100vh - 326px);
-	padding-inline: 5px;
```

Comment:
@paulopmt1 This was introduced in [96892](https://github.com/Automattic/wp-calypso/pull/96892/files#diff-cec006499fa2795f5e135c017ff0d6b25edf9f2e5b2548c54708177a6c452e51R287). But this is breaking alignment in small screens. Could you share your thoughts on this?

<img width="418" alt="image" src="https://github.com/user-attachments/assets/c81897cf-b56d-47a7-91eb-6ac003e44f47" />